eventqueue/processor_test.go

```diff
@@ -53,31 +52,14 @@ func TestProcessor(t *testing.T) {
 	assertNoExecutedItem := func(t *testing.T) {
 		t.Helper()
 
-		// The signal is sent in a background goroutine, so we need to use a wall clock here
-		runtime.Gosched()
 		select {
 		case r := <-executeCh:
 			t.Fatalf("received unexpected item: %s", r.Name)
-		case <-time.After(500 * time.Millisecond):
+		case <-time.After(100 * time.Millisecond):
 			// all good
 		}
 	}
```

Comment:
The reason for using 500ms, as well as `runtime.Gosched()` (which could be kept anyways? it shouldn't cause issues) is that our Actions runner as SLOW... I am concerned that having a timeout that's too low could cause false positives.

Reply:
Fair enough, I'll revert it back.